pi5/pinctrl_utils.go

```diff
@@ -95,21 +109,44 @@ func getRegAddr(childNodePath string, numPAddrCells uint32) (uint64, error) {
 
 	regByteContents, err := os.ReadFile(childNodePath)
 	if err != nil {
-		return uint64(math.NaN()), fmt.Errorf("trouble getting reg addr info for %s: %w\n", childNodePath, err)
+		return INVALID_ADDR, fmt.Errorf("trouble getting reg addr info for %s: %w\n", childNodePath, err)
```

Comment:
still seems like some of the changes from main are not here (thought this was changed to camelcase?). I will help you merge with main once this is approved.